logfire-api/logfire_api/_internal/main.pyi

```diff
@@ -932,7 +932,9 @@ class LogfireSpan(ReadableSpan):
     @property
     def message_template(self) -> str | None: ...
     @property
-    def tags(self) -> Sequence[str]: ...
+    def tags(self) -> tuple[str, ...]: ...
+    @tags.setter
+    def tags(self, new_tags: Sequence[str]) -> None: ...
```

Comment:
The actual class in `logfire_api/__init__.py` needs updating so that setting tags does nothing rather than raising an error, and this needs testing.

Reply:
My bad! I've now updated the `__init__.py` and `__init__.pyi` files to account for the new setter and signature changes.